diff --git a/CMakeLists.txt b/CMakeLists.txt
index 46e884ec..a0b2821e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -44,7 +44,7 @@ find_package(dk2nugenie REQUIRED EXPORT)
 find_package(dk2nudata REQUIRED EXPORT)
 find_package(ppfx REQUIRED EXPORT)
 
-find_package(Boost REQUIRED COMPONENTS headers math_tr1 REQUIRED EXPORT) # boost::math::policies, boost::math::ellint*
+find_package(Boost REQUIRED)
 find_package(CLHEP COMPONENTS Evaluator Geometry Random Vector REQUIRED EXPORT)
 find_package(CRY REQUIRED EXPORT) # find module in nutools
 find_package(GENIE REQUIRED EXPORT) # find module in nugen
