diff -u spack-src.orig/xoap/filter/src/common/MessageFilter.cc spack-src/xoap/filter/src/common/MessageFilter.cc
--- spack-src.orig/xoap/filter/src/common/MessageFilter.cc      2024-05-07 02:43:18.000000000 -0500
+++ spack-src/xoap/filter/src/common/MessageFilter.cc   2024-05-10 17:49:04.666390412 -0500
@@ -45,7 +45,13 @@
 #include "toolbox/utils.h"
 #include "xoap/SOAPElement.h"

-XALAN_CPP_NAMESPACE_USE
+using namespace XALAN_CPP_NAMESPACE;
+#ifndef XALAN_USING_XALAN
+#define XALAN_USING_XALAN(class) using XALAN_CPP_NAMESPACE::class;
+#endif
+#ifndef XALAN_USING_XERCES
+#define XALAN_USING_XERCES(class) using XERCES_CPP_NAMESPACE::class;
+#endif

 xoap::filter::MessageFilter::MessageFilter(const std::string& filterExpression)
 {
diff -u spack-src.orig/xoap/filter/src/common/Platform.cc spack-src/xoap/filter/src/common/Platform.cc
--- spack-src.orig/xoap/filter/src/common/Platform.cc   2024-05-07 02:43:18.000000000 -0500
+++ spack-src/xoap/filter/src/common/Platform.cc        2024-05-10 17:45:33.588781183 -0500
@@ -13,6 +13,10 @@
 #include "xoap/filter/Platform.h"
 #include "xalanc/XPath/XPathEvaluator.hpp"

+#ifndef XALAN_USING_XALAN
+#define XALAN_USING_XALAN(class) using XALAN_CPP_NAMESPACE::class;
+#endif
+
 bool xoap::filter::Platform::initialized_ = false;

 void xoap::filter::Platform::initialize()
