--- a/update-dirt.sh	2021-01-28 14:47:49.818716672 -0600
+++ b/update-dirt.sh	2021-01-28 14:47:06.908767412 -0600
@@ -4,6 +4,14 @@
 
 if ! git rev-parse --show-toplevel > /dev/null 2>&1; then
     echo "Unable to update dirt.sh" 1>&2
+    ds=`date +%Y%m%d`
+    find . -name dirt.sh.in -print |
+       while read f
+       do
+          out=`echo $f | sed -e 's/.in$//'`
+          sed -e 's/@DIRT_BRANCH_ID@/60/' -e "s/@DIRT_TIMESTAMP@/$ds/" < $f > $out
+       done
+    exit
 fi
 
 for d in $(find . -name dirt.sh.in); do
--- a/configure.ac	2014-09-08 15:04:47.000000000 -0500
+++ b/configure.ac	2021-01-28 15:43:30.387913557 -0600
@@ -16,7 +16,6 @@
 LT_CONFIG_LTDL_DIR([libltdl])
 LT_INIT([dlopen win32-dll])
 LTDL_INIT([recursive installable])
-AC_CONFIG_FILES([libltdl/Makefile])
 
 AM_PROG_CC_C_O
 AC_PROG_CC_C99
--- a/configure.ac	2021-01-28 16:02:21.930853425 -0600
+++ b/configure.ac	2021-01-28 16:02:55.201263514 -0600
@@ -12,7 +12,6 @@
 AC_SUBST([MYPROXY_VERSION], [myproxy_version])
 AC_SUBST([GSI_OPENSSH_VERSION], [gsissh_version])
 AM_INIT_AUTOMAKE([1.11 foreign parallel-tests tar-pax subdir-objects])
-AC_CONFIG_HEADERS([ltdlconfig.h])
 LT_CONFIG_LTDL_DIR([libltdl])
 LT_INIT([dlopen win32-dll])
 LTDL_INIT([recursive installable])
