--- spack-src/cmake/modules/FindXROOTD.cmake.orig	2021-02-03 12:07:44.994800000 -0600
+++ spack-src/cmake/modules/FindXROOTD.cmake	2021-02-03 12:07:18.614800000 -0600
@@ -41,6 +41,7 @@
 # -----------------------------------------------------
 find_path(XROOTD_INCLUDE_DIR
     NAMES XrdVersion.hh
+    PATH_SUFFIXES xrootd include/xrootd
     HINTS ${XROOTD_LOCATION} ${XROOTD_LOCATION}/include ${XROOTD_LOCATION}/include/* 
           ${XROOTD_LOCATION}/src/
           /usr/include/xrootd ${CMAKE_INSTALL_PREFIX}/include/xrootd
--- spack-src/cmake/modules/FindGTEST.cmake	2020-12-04 10:10:55.000000000 -0600
+++ /dev/null	2020-12-16 09:56:17.673000000 -0600
@@ -1,60 +0,0 @@
-#
-# This module detects if GTEST is installed and determines where the
-# include files and libraries are.
-#
-# This code sets the following variables:
-# 
-# GTEST_LIBRARIES       = full path to the GTEST libraries
-# GTEST_SSL_LIBRARIES   = full path to the GTEST ssl libraries
-# GTEST_INCLUDE_DIR     = include dir to be used when using the GTEST library
-# GTEST_WSDL2H          = wsdl2h binary
-# GTEST_SOAPCPP2        = soapcpp2 binary
-# GTEST_FOUND           = set to true if GTEST was found successfully
-#
-# GTEST_LOCATION
-#   setting this enables search for GTEST libraries / headers in this location
-
-
-
-
-# -----------------------------------------------------
-# GTEST Libraries
-# -----------------------------------------------------
-find_library(GTEST_LIBRARIES
-    NAMES gtest
-    HINTS ${GTEST_LOCATION}/lib ${GTEST_LOCATION}/lib64 
-          ${GTEST_LOCATION}/lib32
-    DOC "The main GTEST library"
-)
-
-# -----------------------------------------------------
-# GTEST Libraries
-# -----------------------------------------------------
-find_library(GTEST_MAIN_LIBRARIES
-    NAMES gtest_main
-    HINTS ${GTEST_LOCATION}/lib ${GTEST_LOCATION}/lib64 
-          ${GTEST_LOCATION}/lib32
-    DOC "The main GTEST main library"
-)
-
-# -----------------------------------------------------
-# GTEST Include Directories
-# -----------------------------------------------------
-find_path(GTEST_INCLUDE_DIR 
-    NAMES gtest.h
-    HINTS ${GTEST_LOCATION} ${GTEST_LOCATION}/include ${GTEST_LOCATION}/include/gtest ${GTEST_LOCATION}/include/* /usr/include/gtest
-    DOC "The GTEST include directory"
-)
-
-SET(GTEST_DEFINITIONS "")
-
-
-
-# -----------------------------------------------------
-# handle the QUIETLY and REQUIRED arguments and set GTEST_FOUND to TRUE if 
-# all listed variables are TRUE
-# -----------------------------------------------------
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(GTEST DEFAULT_MSG GTEST_LIBRARIES GTEST_MAIN_LIBRARIES 
-    GTEST_INCLUDE_DIR)
-mark_as_advanced(GTEST_INCLUDE_DIR GTEST_LIBRARIES )
--- spack-src/test/CMakeLists.txt.orig	2021-02-03 12:29:21.608428000 -0600
+++ spack-src/test/CMakeLists.txt	2021-02-03 12:30:22.728950000 -0600
@@ -13,7 +13,7 @@
 endif (ONLY_TESTS)
 
 find_package(GTEST)
-include_directories(${GTEST_INCLUDE_DIR})
+include_directories(GTest::gtest)
 
 execute_process(COMMAND cmake -E copy_directory ${CMAKE_CURRENT_SOURCE_DIR}/conf_test  ${CMAKE_CURRENT_BINARY_DIR}/conf_test) 
 
--- spack-src/src/core/file/gfal_file_api.h.orig	2021-02-03 13:53:47.819199000 -0600
+++ spack-src/src/core/file/gfal_file_api.h	2021-02-03 13:52:53.343602000 -0600
@@ -41,7 +41,7 @@
 #if defined __GLIBC_PREREQ && __GLIBC_PREREQ(2,27)
 #include <sys/xattr.h>
 #else
-#include <attr/xattr.h>
+#include <sys/xattr.h>
 #endif
 #endif
 
--- spack-src/src/plugins/file/gfal_file_plugin_main.c.orig	2021-02-03 13:53:07.637602000 -0600
+++ spack-src/src/plugins/file/gfal_file_plugin_main.c	2021-02-03 13:53:09.080199000 -0600
@@ -32,7 +32,7 @@
 #if defined __GLIBC_PREREQ && __GLIBC_PREREQ(2,27)
 #include <sys/xattr.h>
 #else
-#include <attr/xattr.h>
+#include <sys/xattr.h>
 #endif
 #endif
 #include <zlib.h>
--- spack-src/src/core/posix/gfal_posix_api.h.orig	2021-02-03 14:11:56.438436000 -0600
+++ spack-src/src/core/posix/gfal_posix_api.h	2021-02-03 14:12:13.346195000 -0600
@@ -43,7 +43,7 @@
 #if defined __GLIBC_PREREQ && __GLIBC_PREREQ(2,27)
 #include <sys/xattr.h>
 #else
-#include <attr/xattr.h>
+#include <sys/xattr.h>
 #endif
 #endif
 
--- spack-src/src/plugins/lfc/gfal_lfc.c.orig	2021-02-03 14:33:43.831105000 -0600
+++ spack-src/src/plugins/lfc/gfal_lfc.c	2021-02-03 14:33:59.788105000 -0600
@@ -31,7 +31,7 @@
 #if defined __GLIBC_PREREQ && __GLIBC_PREREQ(2,27)
 #include <sys/xattr.h>
 #else
-#include <attr/xattr.h>
+#include <sys/xattr.h>
 #endif
 #endif
 
