diff --git a/CMakeLists.txt b/CMakeLists.txt
index 97428654..bf34661e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,8 +16,8 @@
 cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
 
 set(${PROJECT_NAME}_CMAKE_PROJECT_VERSION_STRING 09.93.01.01)
-find_package(cetmodules REQUIRED)
-project(sbndcode LANGUAGES CXX)
+find_package(cetmodules 2.29.07 REQUIRED)
+project(sbndcode VERSION 09.93.01.01 LANGUAGES CXX)
 
 # for CI:
 # project(sbndcode VERSION 09.91.02.02)
@@ -87,11 +87,13 @@ find_package( Eigen3 REQUIRED )
 find_package( Geant4 REQUIRED )
 find_package( Boost COMPONENTS system REQUIRED )
 find_package( ROOT REQUIRED )
+find_package( hep_hpc REQUIRED)
+find_package( hdf5 REQUIRED)
 
-if(DEFINED ENV{HEP_HPC_DIR} )
-  find_ups_product( hdf5 )
-  find_ups_product( hep_hpc )
-endif()
+# if(DEFINED ENV{HEP_HPC_DIR} )
+#   find_ups_product( hdf5 )
+#   find_ups_product( hep_hpc )
+# endif()
 
 # macros for dictionary and simple_plugin
 include(ArtDictionary)
diff --git a/test/ci/CMakeLists.txt b/test/ci/CMakeLists.txt
index 738b9c02..649f9461 100644
--- a/test/ci/CMakeLists.txt
+++ b/test/ci/CMakeLists.txt
@@ -51,4 +51,4 @@ install_scripts(AS_TEST)
 
 install_fhicl()
 
-install(FILES fcl_file_checks.list DESTINATION "${sbndcode_UPS_PRODUCT_NAME}/${sbndcode_UPS_PRODUCT_VERSION}/test")
+install(FILES fcl_file_checks.list DESTINATION "$ENV{SBNDCODE_DIR}/sbndcode/test")
