diff --git a/CMakeLists.txt b/CMakeLists.txt
index c1faaeb5..9533e55c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -43,7 +43,7 @@ cet_set_compiler_flags(DIAGS CAUTIOUS
 cet_report_compiler_flags(REPORT_THRESHOLD VERBOSE)
 
 #these are minimum required versions,not thea ctual product versions
-find_package(larsoft REQUIRED )
+# find_package(larsoft REQUIRED )
 find_package(libwda REQUIRED )
 find_package(PostgreSQL REQUIRED )
 find_package(Eigen3 REQUIRED )
@@ -80,7 +80,7 @@ find_package(ifdh_art REQUIRED )
 find_package(ROOT REQUIRED )
 find_package(Boost COMPONENTS system program_options REQUIRED)
 find_package(PandoraSDK REQUIRED )
-find_package(PandoraMonitoring REQUIRED )
+# find_package(PandoraMonitoring REQUIRED )
 find_package(SQLite3 REQUIRED )
 find_package(artdaq_core REQUIRED )
 find_package(sbndaq_artdaq_core REQUIRED )
@@ -88,8 +88,8 @@ find_package(TRACE REQUIRED )
 find_package(Microsoft.GSL HINTS $ENV{GUIDELINE_SL_DIR} REQUIRED EXPORT)
 
 find_package(FFTW3f REQUIRED )
-find_package(FFTW3q REQUIRED )
-find_package(FFTW3l REQUIRED )
+# find_package(FFTW3q REQUIRED )
+# find_package(FFTW3l REQUIRED )
 
 # macros for dictionary and simple_plugin
 include(ArtDictionary)
