diff --git a/CMakeLists.txt b/CMakeLists.txt
index 64d84b5..c20d25f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,6 +23,7 @@ cet_set_compiler_flags(DIAGS CAUTIOUS
 cet_report_compiler_flags(REPORT_THRESHOLD VERBOSE)
 
 find_package(nusimdata REQUIRED EXPORT)
+find_package(dk2nudata REQUIRED EXPORT)
 
 find_package(canvas_root_io REQUIRED EXPORT)
 find_package(canvas REQUIRED EXPORT)
diff --git a/lardataobj/AnalysisBase/CMakeLists.txt b/lardataobj/AnalysisBase/CMakeLists.txt
index 4395d9a..f5d67c5 100644
--- a/lardataobj/AnalysisBase/CMakeLists.txt
+++ b/lardataobj/AnalysisBase/CMakeLists.txt
@@ -17,6 +17,7 @@ cet_make_library(SOURCE
   lardataobj::RecoBase
   larcoreobj::SimpleTypesAndConstants
   nusimdata::SimulationBase
+  dk2nu::Tree
   canvas::canvas
   cetlib_except::cetlib_except
 )
diff --git a/lardataobj/MCBase/CMakeLists.txt b/lardataobj/MCBase/CMakeLists.txt
index 1497204..703121a 100644
--- a/lardataobj/MCBase/CMakeLists.txt
+++ b/lardataobj/MCBase/CMakeLists.txt
@@ -10,6 +10,7 @@ cet_make_library(SOURCE
   LIBRARIES
   PUBLIC
   nusimdata::SimulationBase
+  dk2nu::Tree
   ROOT::Physics
 )
 
diff --git a/lardataobj/Simulation/CMakeLists.txt b/lardataobj/Simulation/CMakeLists.txt
index 7c1213b..b51c6ec 100644
--- a/lardataobj/Simulation/CMakeLists.txt
+++ b/lardataobj/Simulation/CMakeLists.txt
@@ -8,6 +8,7 @@ cet_make_library(LIBRARY_NAME GeneratedParticleInfo INTERFACE
   SOURCE GeneratedParticleInfo.h
   LIBRARIES INTERFACE
   nusimdata::SimulationBase
+  dk2nu::Tree
 )
 
 cet_make_library(SOURCE
