diff --git a/sbnanaobj/StandardRecord/Flat/CMakeLists.txt b/sbnanaobj/StandardRecord/Flat/CMakeLists.txt
index 968686a..935f5dc 100644
--- a/sbnanaobj/StandardRecord/Flat/CMakeLists.txt
+++ b/sbnanaobj/StandardRecord/Flat/CMakeLists.txt
@@ -15,7 +15,7 @@ cet_make_library(LIBRARY_NAME sbnanaobj_StandardRecordFlat
                  LIBRARIES    ROOT::TreePlayer
                  )
 if (DEFINED ENV{MRB_BUILDDIR})
-  install_headers(EXTRAS $ENV{MRB_BUILDDIR}/sbnanaobj/sbnanaobj/StandardRecord/Flat/FlatRecord.h $ENV{MRB_BUILDDIR}/sbnanaobj/sbnanaobj/StandardRecord/Flat/FwdDeclare.h)
+  install_headers(EXTRAS $ENV{MRB_BUILDDIR}/sbnanaobj/StandardRecord/Flat/FlatRecord.h $ENV{MRB_BUILDDIR}/sbnanaobj/StandardRecord/Flat/FwdDeclare.h)
 else()
   install_headers(EXTRAS ${CMAKE_BINARY_DIR}/sbnanaobj/StandardRecord/Flat/FlatRecord.h ${CMAKE_BINARY_DIR}/sbnanaobj/StandardRecord/Flat/FwdDeclare.h)
 endif()
diff --git a/sbnanaobj/StandardRecord/Proxy/CMakeLists.txt b/sbnanaobj/StandardRecord/Proxy/CMakeLists.txt
index 587cddf..379c25f 100644
--- a/sbnanaobj/StandardRecord/Proxy/CMakeLists.txt
+++ b/sbnanaobj/StandardRecord/Proxy/CMakeLists.txt
@@ -17,7 +17,7 @@ cet_make_library(LIBRARY_NAME sbnanaobj_StandardRecordProxy
                  SOURCE       SRProxy.cxx Instantiations.cxx
                  LIBRARIES    ROOT::TreePlayer)
 if (DEFINED ENV{MRB_BUILDDIR})
-  install_headers(EXTRAS $ENV{MRB_BUILDDIR}/sbnanaobj/sbnanaobj/StandardRecord/Proxy/SRProxy.h $ENV{MRB_BUILDDIR}/sbnanaobj/sbnanaobj/StandardRecord/Proxy/FwdDeclare.h)
+  install_headers(EXTRAS $ENV{MRB_BUILDDIR}/sbnanaobj/StandardRecord/Proxy/SRProxy.h $ENV{MRB_BUILDDIR}/sbnanaobj/StandardRecord/Proxy/FwdDeclare.h)
 else()
   install_headers(EXTRAS ${CMAKE_BINARY_DIR}/sbnanaobj/StandardRecord/Proxy/SRProxy.h ${CMAKE_BINARY_DIR}/sbnanaobj/StandardRecord/Proxy/FwdDeclare.h)
 endif()

