--- gfal2-bindings-v1.10.0/CMakeLists.txt	2020-12-04 11:30:04.000000000 -0600
+++ spack-src/CMakeLists.txt	2021-02-10 14:28:05.477349000 -0600
@@ -25,35 +25,8 @@
 
 message("Python: ${PYTHON_EXECUTABLE}")
 
-if (PYTHON_EXECUTABLE)
-    if(RHEL STREQUAL "6")
-      find_package(Boost COMPONENTS python REQUIRED)
-    elseif(RHEL STREQUAL "7")
-      find_package(Boost COMPONENTS python REQUIRED)
-    else()
-      find_package(Boost COMPONENTS python27 REQUIRED)
-    endif()
-endif(PYTHON_EXECUTABLE)
-
-
-if (PYTHON3_CURRENT_VERSION)
-    message("Python 3 installed! ${PYTHON3_CURRENT_VERSION}")
-
-    string(REPLACE "." "" Boost_PYTHON_3_SUFFIX "${PYTHON3_CURRENT_VERSION}")
-
-    find_library(Boost_PYTHON_3_LIBRARY
-        NAMES boost_python3 boost_python${Boost_PYTHON_3_SUFFIX}  boost_python-py${Boost_PYTHON_3_SUFFIX}
-        HINTS
-            ${Boost_LIBRARYDIR}
-            "/usr/lib"
-            "/usr/lib64"
-    )
-    if (Boost_PYTHON_3_LIBRARY)
-        message("Boost Python3 library found! ${Boost_PYTHON_3_LIBRARY}")
-    else()
-        message("Boost Python3 library not found")
-    endif ()
-endif ()
+string(REPLACE "." "" Boost_PYTHON_SUFFIX "${PYTHON3_CURRENT_VERSION}")
+find_package(Boost COMPONENTS python${Boost_PYTHON_SUFFIX}  REQUIRED)
 
 #define PROJECT vars
 set(PROJECT_NAME_MAIN "gfal2-python")
